src/js/widgets/TextWidget.jsx

```diff
@@ -7,14 +8,15 @@ export default function TextWidget(props) {
   if (!inputType) {
     inputType = numberTypes.has(props.schema.type) ? 'number' : props.type;
   }
+  const widgetClasses = classnames(props.options.widgetClassNames);
```

Comment:
This could be done inline below, without creating a variable. It probably doesn't matter as far as performance is concerned but I have a slight preference for putting it below unless we need widgetClasses in two places.